<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Three Brooks</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/ThreeBrooks/app.svg</normaloff>:/ThreeBrooks/app.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewRecipe"/>
    <addaction name="actionOpenRecipe"/>
    <addaction name="separator"/>
    <addaction name="actionCloseRecipe"/>
    <addaction name="separator"/>
    <addaction name="actionSaveRecipe"/>
    <addaction name="actionSaveAsRecipe"/>
    <addaction name="separator"/>
    <addaction name="actionPrintRecipe"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionReloadStyleSheet"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt4"/>
    <addaction name="actionAboutThreeBrooks"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionHideSidebar"/>
    <addaction name="separator"/>
    <addaction name="actionRecipeRight"/>
    <addaction name="actionRecipeLeft"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenRecipe">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/open.svg</normaloff>:/ThreeBrooks/Recipe/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open Recipe...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveRecipe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/save.svg</normaloff>:/ThreeBrooks/Recipe/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save Recipe...</string>
   </property>
   <property name="toolTip">
    <string>Save the current recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewRecipe">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/create.svg</normaloff>:/ThreeBrooks/Recipe/create.svg</iconset>
   </property>
   <property name="text">
    <string>New Recipe...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSaveAsRecipe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/saveAs.svg</normaloff>:/ThreeBrooks/Recipe/saveAs.svg</iconset>
   </property>
   <property name="text">
    <string>Save Recipe As...</string>
   </property>
   <property name="toolTip">
    <string>Save the current recipe As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCloseRecipe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/close.svg</normaloff>:/ThreeBrooks/Recipe/close.svg</iconset>
   </property>
   <property name="text">
    <string>Close Recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/appExit.svg</normaloff>:/ThreeBrooks/appExit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReloadStyleSheet">
   <property name="text">
    <string>Reload StyleSheet</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPrintRecipe">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/print.svg</normaloff>:/ThreeBrooks/Recipe/print.svg</iconset>
   </property>
   <property name="text">
    <string>Print Recipe</string>
   </property>
   <property name="toolTip">
    <string>Print recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/documentation.svg</normaloff>:/ThreeBrooks/documentation.svg</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionAboutQt4">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/qt.svg</normaloff>:/ThreeBrooks/qt.svg</iconset>
   </property>
   <property name="text">
    <string>About Qt4...</string>
   </property>
  </action>
  <action name="actionAboutThreeBrooks">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/about.svg</normaloff>:/ThreeBrooks/about.svg</iconset>
   </property>
   <property name="text">
    <string>About Three Brooks...</string>
   </property>
  </action>
  <action name="actionRecipeRight">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/right.svg</normaloff>:/ThreeBrooks/Recipe/right.svg</iconset>
   </property>
   <property name="text">
    <string>Next Recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionRecipeLeft">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/Recipe/left.svg</normaloff>:/ThreeBrooks/Recipe/left.svg</iconset>
   </property>
   <property name="text">
    <string>Last Recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Backtab</string>
   </property>
  </action>
  <action name="actionHideSidebar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/ThreeBrooks/hideSidebar.svg</normaloff>:/ThreeBrooks/hideSidebar.svg</iconset>
   </property>
   <property name="text">
    <string>Ingredient Picker</string>
   </property>
   <property name="toolTip">
    <string>Hide the ingredient list picker</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
